### PR TITLE
add onOpen event handler

### PR DESCRIPTION
**Pull request by bschaufler**

### CODE REVIEW

File: addon/components/g-map-infowindow.js [hunk 1]
```diff
@@ -85,7 +89,14 @@ const GMapInfowindowComponent = Ember.Component.extend({
       this.sendAction('onClose');
     }
   },
-
+  handleOpenClickEvent() {
+  const { onOpen } = this.attrs;
+  if (typeOf(onOpen) === 'function') {
+    onClose();
```
Comment (asennikov) [inline addon/components/g-map-infowindow.js]: looks like it should be `onOpen` here

File: addon/components/g-map-infowindow.js [hunk 2]
```diff
@@ -85,7 +89,14 @@ const GMapInfowindowComponent = Ember.Component.extend({
...
+  } else {
+    this.sendAction('onOpen', this.context);
```
Comment (asennikov) [inline addon/components/g-map-infowindow.js]: `context`?

**Comment by asennikov** [comment]
Could you also check failed styling checks in travis?
Also, would be really cool if you could add a few tests for the new event handler, similar to what we already have for other events.

**Comment by steveszc** [comment]
close in favor of #100

**Pull request closed without merging**